cas-server-documentation/installation/RADIUS-Authentication.md

```diff
@@ -7,52 +7,47 @@ title: CAS - RADIUS Authentication
 RADIUS support is enabled by including the following dependency in the Maven WAR overlay:
 
 {% highlight xml %}
-    <dependency>
-      <groupId>org.jasig.cas</groupId>
-      <artifactId>cas-server-support-radius</artifactId>
-      <version>${cas.version}</version>
-    </dependency>
+<dependency>
+  <groupId>org.jasig.cas</groupId>
+  <artifactId>cas-server-support-radius</artifactId>
+  <version>${cas.version}</version>
+</dependency>
 {% endhighlight %}
 
-## RADIUS Components
+## Configuration
 
-######`RadiusAuthenticationHandler`
-The RADIUS handler accepts username/password credentials and delegates authentication to one or more RADIUS
-servers. It supports two types of failovers: failover on an authentication failure, and failover on a server exception.
-
-* `failoverOnAuthenticationFailure` - True to continue to the next configured RADIUS server on authentication failure,
-false otherwise. This flag is typically set when user accounts are spread across one or more RADIUS servers.
-* `failoverOnException` - True to continue to next configured RADIUS server on an error other than authentication
-failure, false otherwise. This flag is typically set to support highly available deployments where authentication
-should proceed in the face of one or more RADIUS server failures.
-* `servers` - Array of RADIUS servers to delegate to for authentication.
-
-
-######`JRadiusServerImpl`
-Component representing a RADIUS server has the following configuration properties.
+{% highlight xml %}
+<bean id="radiusAuthenticationHandler"
+      class="org.jasig.cas.adaptors.radius.authentication.handler.support.RadiusAuthenticationHandler"
+      p:failoverOnAuthenticationFailure="${cas.radius.failover.authn:false}"
+      p:failoverOnException="${cas.radius.failover.exception:false}"
+      p:servers-ref="radiusServers" />
 
-* `protocol` - radius protocol to use.
-* `clientFactory` - factory establish and create radius client instances.
+<util:list id="radiusServers">
+     <ref bean="radiusServer" />
+</util:list>
 
+{% endhighlight %}
 
-## RADIUS Configuration Example
-{% highlight xml %}
-<bean id="radiusServer"
-      class="org.jasig.cas.adaptors.radius.JRadiusServerImpl"
-      c:protocol="EAP_MSCHAPv2"
-      c:clientFactory-ref="radiusClientFactory" />
+The following settings may control the configuration RADIUS server and client:
+
+{% highlight properties %}
+# cas.radius.client.inetaddr=localhost
+# cas.radius.client.port.acct=
+# cas.radius.client.socket.timeout=60
+# cas.radius.client.port.authn=
+# cas.radius.client.sharedsecret=N0Sh@ar3d$ecReT
+# cas.radius.server.protocol=EAP_MSCHAPv2
```

Comment:
OK, cool. I was too quick look :) In that case, I'd just leave it be, unless a specific request comes up.